monegros-restaurant-dianamonegrodiaz/src/monegros/restaurant/entities/Cashier.java

```diff
@@ -1,11 +1,40 @@
 package monegros.restaurant.entities;
 
 public class Cashier extends Employee {
+	private Order newOrder;
```

Comment:
Bad idea to do this since one cashier can create many orders. That's why orders are made by an user.

Reply:
But, as soon as the cashier gets one order it is send to the Kitchen.